analysis/wwz/make_datacards.py

```diff
@@ -598,7 +606,7 @@ def main():
         handle_per_year_systs_for_fr(yld_dict_mc_allyears,run,do_jec27)
 
     yld_dict_mc = yld_dict_mc_allyears["FR"]
-    yld_dict_data = yt.get_yields(histo,sample_names_dict_data["FR"])
+    yld_dict_data = yt.get_yields(histo,sample_names_dict_data["FR"],blind=False)
```

Comment:
Can you modify this so that it goes with the command line flag and is not just hard coded to false? 

One problem is that flag and this bool are I think opposite. So you might need e.g. a "not" somewhere

But basically we just want this to be turned on and off from command line

Reply:
done